### ISSUE TITLE
Error running stg_dbt__node_executions model

### ISSUE THREAD

**Issue by lisa-nalyvaiko** [brooklyn-data/dbt_artifacts#118]
When I try to run all models by running `dbt build -s dbt_artifacts`, I get the following error:
```
Database Error in model stg_dbt__node_executions (models/staging/stg_dbt__node_executions.sql)
07:36:46    001789 (42601): SQL compilation error:
07:36:46    invalid number of result columns for set operator input branches, expected 12, got 15 in branch 2
07:36:46    compiled SQL at target/run/dbt_artifacts/models/staging/stg_dbt__node_executions.sql
```
When checking the model compiled SQL, I noticed that indeed those two things produce different number of columns, thus UNION can't work:
```
-- V1 uploads
    {{ flatten_results("run_results") }}

    union all

    -- V2 uploads
    -- NB: We can safely select * because we know the schemas are the same
    -- as they're made by the same macro.
    select * from base_v2
```
This is my first attempt of trying to use your package, so I'm not sure whether I'm missing something. I'm also not sure what' V1 uploads sql comment means, since I've only ever used only V2 macro. I'm on 0.8.0 version. 
 
Steps to reproduce:
- run `dbt run-operation create_artifact_resources`
- run `dbt run -m +model_name`
- run `dbt --no-write-json run-operation upload_dbt_artifacts_v2`
- run `dbt build -s dbt_artifacts`

**Comment by NiallRees** [comment]
Hi @lisa-nalyvaiko - thanks for creating this issue. Both `{{ flatten_results("run_results") }}` and `base_v2` should have 12 columns. Would you be able to share the columns which are in each of those?

**Comment by LukasSandm** [comment]
I had the same issue. For me the problem arises due to a migrate from an older version of the package to 0.8.0 and since I created the dbt_manifest_nodes previously. The `dbt_manifest_nodes` table has 23 columns in my case and the run operations does not change that because it is a create table if not exists command, and the table already exists.

I fixed it by manually running the create table commands for `dbt_manifest_nodes` & `dbt_run_results_nodes`, but with `create or replace table`. We are using v1, so these tables were empty for us.
